theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -197,11 +198,10 @@ const CurveEditorPopover: React.FC<IProps> = (props) => {
 
   // When `preview` or `edit` change, use the `tempTransaction` to change the
   // curve in Theate's data.
-  useMemo(
-    () =>
-      setTempValue(tempTransaction, props, cur, next, preview ?? edit ?? ''),
-    [preview, edit],
-  )
+  useMemo(() => {
```

Comment:
Catching this just now. This should be a `useEffect()` rather than `useMemo()` as it's not returning any value and is running a side-effect.

I'd use `useLayoutEffect()` to ensure no delay between first render and the effect.